packages/react/src/auto/polaris/PolarisAutoTable.tsx

```diff
@@ -46,15 +59,51 @@ export const PolarisAutoTable = <
 ) => {
   const { onClick } = props;
 
-  const [{ rows, columns, metadata, fetching, error, page, search }, refresh] = useTable<GivenOptions, SchemaT, FinderFunction, Options>(
-    props.model,
-    {
-      select: props.select,
-      columns: props.columns,
-      pageSize: props.pageSize,
-      live: props.live,
-    } as any
+  const [{ rows, columns, metadata, fetching, error, page, search, sort }, refresh] = useTable<
+    GivenOptions,
+    SchemaT,
+    FinderFunction,
+    Options
+  >(props.model, {
+    select: props.select,
+    columns: props.columns,
+    pageSize: props.pageSize,
+    live: props.live,
+    sort: props.sort,
+    filter: props.filter,
+  } as any);
+
+  const [sortColumnIndex, setSortColumnIndex] = useState<number | undefined>(
+    columns ? getColumnIndex(columns, props.sort ? Object.keys(props.sort)[0] : undefined) : undefined
+  );
+  const [defaultColumnIndex, setDefaultColumnIndex] = useState(true);
+  const [sortDirection, setSortDirection] = useState<"ascending" | "descending" | undefined>(
```

Comment:
Instead of doing the 'Ascending' => 'ascending' mapping here in the state, I think we could just do it right before we pass the value to polaris

like: defaultSortDirection={gadgetToPolarisSortKey(defaultSortDirection)}